### PR TITLE
Handling connection timeout for download request

### PR DESCRIPTION
**Pull request by stulees**
- Setting a 5 minute connection timeout on the download request
- Unit test added

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks

File: config/config.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ module.exports = {
   logFilePath: path.join(riseCachePath,"risecache.log"),
   diskThreshold: 500 * 1024 * 1024,  // 500 MB
   fileUpdateDuration: 1200000, // 20 minutes,
+  requestTimeout: 300000, // 5 minutes
```
Comment (donnapep) [inline config/config.js]: @stulees Last I hear this was supposed to be 120000. Has that changed?

Comment (rodrigopavezi) [inline config/config.js]: @stulees same question. cheers

Comment (stulees) [inline config/config.js]: @donnapep It seems a really long time to allow for and I don't remember the discussion on deciding 20 mins, so I opted for 5. Was there a reason for 20 mins?

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Timeout changed to 20 mins in last commit.

**Comment by donnapep** [comment]
@stulees I think it's supposed to be 2 as Rodrigo mentioned, so 120000 instead of 1200000.

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Changed to 2 mins. Please review, thanks

**Pull request merged**